data/data/tools.go

```diff
@@ -0,0 +1,12 @@
+package data
+
+import "embed"
+
+// This file and the package it creates are intended to distribute the static
+// assets of this data directory in the go module, specifically for consumption
+// by the ARO wrapper on a temporary basis. The installer itself does not import
+// this package and no other codebases should either, as we intend to remove it
+// once ARO has moved off the wrapper.
+
+//go:embed bootstrap/* manifests/*
```

Comment:
are these two sufficient?

Reply:
I expect we will need `coreos`.
The directory `bootstrap/files/root/.docker` starts with a `.` so we need to force that to be included.
```suggestion
//go:embed all:bootstrap manifests coreos
```

Reply:
Hmm, we didn't actually do this did we